node/lib/task.ts

```diff
@@ -511,7 +511,11 @@ export function stats(path: string): FsStats {
  * @returns   boolean 
  */
 export function exist(path: string): boolean {
-    return path && fs.existsSync(path);
+    try {
+        return path && fs.statSync(path) != null;
+    } catch (err) {
+        return false;
```

Comment:
Do not swallow exceptions.  You've just masked whatever the exceptional case is and returned a conclusive answer which s not necessarily correct.